diff --git a/node_modules/react-native-star-rating/StarRating.js b/node_modules/react-native-star-rating/StarRating.js
index 7aecc95..25cd114 100644
--- a/node_modules/react-native-star-rating/StarRating.js
+++ b/node_modules/react-native-star-rating/StarRating.js
@@ -1,8 +1,9 @@
 // React and react native imports
 import React, { Component } from 'react';
-import { View, ViewPropTypes, StyleSheet } from 'react-native';
+import { View, StyleSheet } from 'react-native';
 import PropTypes from 'prop-types';
 import { View as AnimatableView } from 'react-native-animatable';
+import { ViewPropTypes } from 'deprecated-react-native-prop-types';
 
 // Local file imports
 import StarButton from './StarButton';
@@ -76,7 +77,7 @@ const defaultProps = {
   reversed: false,
   starSize: 40,
   starStyle: {},
-  selectedStar: () => {},
+  selectedStar: () => { },
 };
 
 class StarRating extends Component {
